puppet/modules/kubernetes/manifests/apiserver.pp

```diff
@@ -100,8 +112,11 @@
     $_oidc_signing_algs = []
   }
 
-  # Do not insecure bind the API server on kubernetes 1.6+
-  $insecure_port = $::kubernetes::_apiserver_insecure_port
+  # Do not insecure bind the API server on kubernetes 1.11+
```

Comment:
Should be `Do not set insecure_port variable of the API server on kubernetes 1.11+`